models/tcp_route_test.go

```diff
@@ -105,6 +105,31 @@ var _ = Describe("TCP Route", func() {
 					Expect(tcpRouteMapping2.Matches(tcpRouteMapping)).To(BeFalse())
 				})
 			})
+
+			Context("when two routes have different", func() {
```

Comment:
when two routes have different what? 😄

Reply:
bringing the conversation to https://github.com/cloudfoundry/routing-api/pull/75#discussion_r2223676098